test/timings.ts

```diff
@@ -0,0 +1,50 @@
+import test from 'ava';
+import got from '../source/index.js';
+
+test('http/1 timings', async t => {
+	const {timings} = await got('https://httpbin.org/anything');
```

Comment:
Can you use a local server for this one please? https://github.com/sindresorhus/got/blob/2ac07e1b60f59e2219bd6c2809a9e40f56b146b6/test/http.ts#L35-L41

Reply:
Just for the http/1 test? It seems the `withServer` helper only support http/1.

Would it be feasible to implement the http/2 support within `withServer`? I gave a quick look, and since I saw the implementation for an https server, I guess it would be possible?

Reply:
This is another issue: https://github.com/sindresorhus/got/issues/1093